BuildResidentialHPXML/resources/geometry.rb

```diff
@@ -1607,12 +1567,24 @@ def self.create_single_family_attached(runner:,
     end
     roof_type = geometry_roof_type
     roof_pitch = geometry_roof_pitch
+    left_wall = geometry_unit_left_wall_is_adiabatic
+    right_wall = geometry_unit_right_wall_is_adiabatic
+    front_wall = geometry_unit_front_wall_is_adiabatic
+    back_wall = geometry_unit_back_wall_is_adiabatic
```

Comment:
Can we make these variables more descriptive? E.g. `adiabatic_left_wall` instead of just `left_wall`.